portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,94 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/*
+ In order to access the secret page you have to solve a randomly generated equation.
+ Upon the input of a correct answer a button will appear, linking the user to the page.
+ */
+function secretPage(){
+  //Generates a number between 1 and 200 for both numbers.
+  this.a = Math.floor((Math.random()*200)+100);
+  this.b = Math.floor((Math.random()*200)+100);
+
+
+  //If b is a greater number we flip the values
+  if(this.b > this.a){
+    var hold =this.b;
```

Comment:
nit: Keep spacing consistent

```suggestion
    let hold = this.b;
```

Reply:
Got it